### ISSUE TITLE
Tootlip doesn't get read by Narrator in Firefox

### ISSUE THREAD

**Issue by kroudams** [microsoft/fluentui#21501]
1. Open Firefox (in my case it's Firefox 96.0.2 (64-bit) on Windows 10)
2. Go to the https://developer.microsoft.com/en-us/fluentui#/controls/web/tooltip -> Tooltip on absolutely-positioned element section 3. Turn on Windows Narrator Screen Reader
4. Focus `Hover over me` button. Tooltip will be presented but won't be read by the Narrator.

I've checked quite a few websites that use Fluent UI's Tooltip, on every one of them Tooltip hasn't been read by the Narrator in Firefox.

**Comment by micahgodbolt** [comment]
This is unfortunately something that we are not going to address.

As mentioned in our wiki, Edge is our primary target for Narrator support due to the difference in a11y APIs between browsers. https://github.com/microsoft/fluentui/wiki/Accessibility#screen-reader

Also, in regards to tooltips, your aria-label/descriptions should be sufficient for any non-sighted users, as tooltips expose that information to sighted users.

**Comment by kroudams** [comment]
> Also, in regards to tooltips, your aria-label/descriptions should be sufficient for any non-sighted users, as tooltips expose that information to sighted users.

`aria-describedby` is automatically set by Fluent UI to the tooltip.

![image](https://user-images.githubusercontent.com/87769044/156530452-5427b500-aff8-4020-b99f-524b6543eb92.png)

**Comment by micahgodbolt** [comment]
Good point about described by being set to the tooltip, I'll take back that part of the comment. I still believe our markup is correct. we're using aria-describedby just as indicated in the W3 specification 

https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA1 => Example 4: Using aria-describedby to associate tooltips with form fields

If you have a suggestion for better markup I'm happy to discuss it. But I believe this control is working as expected.

**Comment by kroudams** [comment]
The main problem remains that Narrator doesn't read it, thus making the Tooltip inaccessible.

**Comment by micahgodbolt** [comment]
> The main problem remains that Narrator doesn't read it, thus making the Tooltip inaccessible.

It reads in Edge though, right?. Edge which is our primary Narrartor supported browser.
